External/FEXCore/Source/Interface/Core/Dispatcher/Arm64Dispatcher.cpp

```diff
@@ -543,7 +543,8 @@ void Arm64Dispatcher::EmitDispatcher() {
   ClearICache(reinterpret_cast<void*>(DispatchPtr), End - reinterpret_cast<uint64_t>(DispatchPtr));
 
   if (CTX->Config.BlockJITNaming()) {
-    std::string Name = "Dispatch_" + std::to_string(FHU::Syscalls::gettid());
+    fextl::string Name = "Dispatch_";
+    Name += FHU::Syscalls::gettid();
```

Comment:
Was it intended to drop the `std::to_string` here? Won't this truncate the thread id to a `char` that is appended to the string?

Reply:
Resolved.